### PR TITLE
Files to run the `cluster-cell-calling` module 

### LINKED ISSUE #85: Files to run the `cluster-cell-calling` module
## What are the scientific goals of the analysis?

To add files and README files on how to run the `cluster-cell-calling` module.

### PR DESCRIPTION
**Pull request by AntoniaChroni**
# Purpose/implementation Section

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## What scientific question/module of analysis is your analysis addressing?

Files to run the `cluster-cell-calling` module.

## What GitHub issue does your pull request address?

Closes #85 .

## Directions for reviewers 
Tell potential reviewers what kind of feedback you are soliciting.

### Which areas should receive a particularly close look?

Please review the code and documentation for clarity.

## Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.

## Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/cluster-cell-calling/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+|   ├── Report_find_markers_<Sys.Date()>.html
+|   └── Report_find_markers_<Sys.Date()>.pdf
+├── README.md
+├── results
+|   ├── 01_cluster_cell_calling_{resolution}
+|   └── 02_find_markers
+├── run-cluster-cell-calling-step1.R
+├── run-cluster-cell-calling-step2.R
+├── run-cluster-cell-calling-step1.sh
+├── run-cluster-cell-calling-step2.sh
```
Comment (sharonfreshour) [inline analyses/cluster-cell-calling/README.md:62]: I think the file structure needs to be updated to reflect that there is only the `run-cluster-cell-calling.sh` in the module, not separate `run-cluster-cell-calling-step1.sh` and `run-cluster-cell-calling-step2.sh` files.

Comment (sharonfreshour) [reply analyses/cluster-cell-calling/README.md:62]:
```suggestion
```

File: analyses/cluster-cell-calling/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+|   ├── Report_cluster_cell_calling_{resolution}_<Sys.Date()>.pdf
+|   ├── Report_find_markers_<Sys.Date()>.html
+|   └── Report_find_markers_<Sys.Date()>.pdf
+├── README.md
+├── results
+|   ├── 01_cluster_cell_calling_{resolution}
+|   └── 02_find_markers
+├── run-cluster-cell-calling-step1.R
+├── run-cluster-cell-calling-step2.R
+├── run-cluster-cell-calling-step1.sh
```
Comment (sharonfreshour) [inline analyses/cluster-cell-calling/README.md:61]:
```suggestion
├── run-cluster-cell-calling.sh
```

**Review by sharonfreshour — APPROVED** [review]
Code looks good and logical. I made one comment/suggestion for editing the file structure section of the README.

File: analyses/cluster-cell-calling/README.md (hunk 2 above)
Comment (AntoniaChroni) [reply analyses/cluster-cell-calling/README.md]: Fixed here 96729dff212ab007f73b07edff68bdfec28ef0ba

**Pull request merged**